data/event_scripts.s

```diff
@@ -527,9 +527,9 @@ EventScript_23B5A1:: @ 823B5A1
 	case 127, EventScript_23B66E
```

Comment:
Resolved, and moved to a separate file. It's in the middle of all the map script data, it might be better to move it to data/maps so we don't need secret_base_1 and _2, even though it's not technically a map script file

Reply:
Actually it looks like secret_base_1.inc is a map script file, it's just partially shoved into Yellow Cave 4 and shared across all secret bases. Going to split that out of Yellow Cave 4, then move secret_base_1.inc into data/maps and rename.